src/form/form.tsx

```diff
@@ -99,7 +109,8 @@ interface Props<MODEL> {
     validationOptions?: ValidationOptions;
     resolveViolationMessage?: ViolationMessageResolver;
     additionalButtonRenderers?: Array<(buttonProps: FormButtonProps) => React.ReactNode>;
-    isInitiallyDirty?: boolean;
+    isAlwaysDirty?: boolean;
+    validateVisibleFields?: boolean;
```

Comment:
I think it might make sense to always do that. At least I cannot come up with a case that makes sense of validating a hidden field since nothing can be done to make it valid.

Reply:
I would fully agree if I was brave enough to test all usages of the form :)

Reply:
Done too :D